src/containers/TexterTodoList.jsx

```diff
@@ -178,16 +184,55 @@ export const dataQuery = gql`
           sweepComplete
         }
         allContactsCount: contactsCount
-        unmessagedCount: contactsCount(contactsFilter: $needsMessageFilter)
-        unrepliedCount: contactsCount(contactsFilter: $needsResponseFilter)
-        badTimezoneCount: contactsCount(contactsFilter: $badTimezoneFilter)
+        unmessagedCount: contactsCount(contactsFilter: {
```

Comment:
these are all moving constants in the variables{} section below to actual constant within the gql, which I think is more readable anyway.